packages/story-editor/src/components/colorPicker/karma/colorPicker.karma.js

```diff
@@ -158,7 +158,11 @@ describe('ColorPicker', () => {
 
         // Add text and apply the previously saved color.
         await fixture.events.click(fixture.editor.library.textAdd);
-        await waitFor(() => fixture.editor.canvas.framesLayer.frames[1].node);
+        await waitFor(() => {
+          if (!fixture.editor.canvas.framesLayer.frames[1].node) {
+            throw new Error('node not ready');
+          }
+        });
```

Comment:
I had to fix a bad rebase, so while I was in here I updated the `waitFor`s too be a little more robust and actually work the way we want them too.